src/iOS/Xamarin.iOS/Samples/Augmented Reality/CollectDataAR/readme.md

```diff
@@ -0,0 +1,62 @@
+
+> ⚠️ **WARNING**: collection of photos is completely automatic; consider your surroundings when adding features.
+
+## How it works
+
+1. Create an `ARSceneView`. Set the atmosphere effect to `None` and the space effect to `None`. Create and show a scene in the scene view.
+2. Load the feature service and display it with a feature layer.
+3. Create and add the elevation surface to the scene.
+4. Create a graphics overlay for planning the location of features to add. Configure the graphics overlay with a renderer and add the graphics overlay to the scene view.
+5. When the user taps the screen, use `ARSceneView.ARScreenToLocation` to find the real-world location of the tapped object using ARKit/ARCore plane detection.
+6. Add a graphic to the graphics overlay preview where the feature will be placed and allow the user to visually verify the placement.
+7. When the user presses the button, take the current AR frame from `ARSceneView.ARSCNView.Session.CurrentFrame.CapturedImage`. Rotate the image appropriately and convert it to a JPEG for efficient storage.
+8. Prompt the user for a tree health value, then create the feature. Upon successful creation of the feature, use `feature.AddAttachment` to add the image.
+
+## Relevant API
+
+* ARSceneView
+* GraphicsOverlay
+* SceneView
+* Surface
+
+## About the data
+
+The sample uses a publicly-editable sample [tree survey feature service](https://arcgisruntime.maps.arcgis.com/home/item.html?id=8feb9ea6a27f48b58b3faf04e0e303ed). You can use AR to quickly record the location and health of a tree while seamlessly capturing a photo.
+
+## Additional information
+
+This sample requires a device that is compatible with ARKit 1 on iOS or ARCore 1.8 on Android.
+
+There are two main approaches for identifying the physical location of tapped point:
+
+* **ARSceneView.ARScreenToLocation** - uses plane detection provided by ARKit/ARCore to determine where _in the real world_ the tapped point is.
+* **SceneView.ScreenToLocation** - determines where the tapped point is _in the virtual scene_. This is problematic when the opacity is set to 0 and you can't see where on the scene that is. Real-world objects aren't accounted for by the scene view's calculation to find the tapped location; for example tapping on a tree might result in a point on the basemap many meters away behind the tree.
+
+This sample only uses the `arScreenToLocation` approach, as it is the only way to get accurate positions for features not directly on the ground in real-scale AR.
+
+Note that unlike other scene samples, a basemap isn't shown most of the time, because the real world provides the context. Only while calibrating is the basemap displayed at 50% opacity, to give the user a visual reference to compare to.
+
+**Real-scale AR** is one of three main patterns for working with geographic information in augmented reality. See [Augmented reality]() in the guide for more information.
```

Comment:
Doc link can point to https://developers.arcgis.com/net/latest/ios/guide/display-scenes-in-augmented-reality.htm (note these links are SDK-specific)